core/templates/services/services.constants.ts

```diff
@@ -69,4 +69,7 @@ export class ServicesConstants {
 
   public static RTE_COMPONENT_SPECS = (
       require('rich_text_components_definitions.ts'));
+
+  public static WHITELISTED_IFRAME_SECRETS = [
+    '25b49a7f706348c29c5b3709ee4570bb'];
```

Comment:
Pretty sure this sort of thing should be generated on the fly? Otherwise ... it's not a secret, anyone can look at this codebase and read it.

Reply:
Thought so, talked to Ben about this, and he said that it wasn't a huge deal because there isn't any actual security vulnerability that comes with it. But maybe this needs to be clarified.